gittrigger/gittrigger.go

```diff
@@ -35,12 +35,36 @@ func (g *gitTrigger) getLatestCommit() error {
 	return nil
 }
 
-func (g gitTrigger) checkTrigger() bool {
+func (g *gitTrigger) GetRemote() (string, error) {
+	var out bytes.Buffer
+	cmd := exec.Command("bash", "-c", "git remote | head -n 1")
```

Comment:
I don't think bash is available on OSX anymore by default, it's less likely to be present in containers as well. Won't work on Windows on command.com either (although I guess that's a niche use case). Stealing from your code below (and untested):

```suggestion
	cmd := exec.Command("git", "remote")
	cmd.Stdout = &out
	err := cmd.Run()

	if err != nil {
		return "", err
	}
	remote_name := strings.TrimSpace(strings.Split(strings.ReplaceAll(out.String(), "\r\n", "\n"), "\n")[0])
```